### PR TITLE
B hub skulls

### PR DESCRIPTION
**Pull request by b-hub**
#### Checklist

* [ ] Use a separate branch in your local repo (not `master`).
* [ ] Test coverage is 90% or better (or you have a story for why it's ok).

### CODE REVIEW

**Comment by flamecoals** [comment]
Is this playable and/or ready for review/merging?

**Comment by b-hub** [comment]
@flamecoals still a work in progress. We can close without merging for now and the next pull request I make will be playable and ready for review

**Comment by flamecoals** [comment]
No need to close, I just wanted to make sure you weren't waiting for me :)

File: web/src/games/bombsAndBunnies/CardComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+  render() {
+    return (
+      <div
+        className={[
+          css.Card,
+          this.front,
+          this.props.selectable ? css.Selectable : '',
+          this.props.selected ? css.Selected : '',
+          //this.props.card.showing ? css['CardFront_' + this.props.card.value] : css.hidden,
```
Comment (flamecoals) [inline web/src/games/bombsAndBunnies/CardComponent.tsx:23]: remove comment

**Review by flamecoals — COMMENTED** [review]
This is overall looking really good! Just need to make sure we can increase the test coverage for the rules.

According to our new contributing guidelines, we can merge this PR without publishing it, even though it might not be fully ready yet. Then, you can send smaller PRs and we can publish when the game has all the attributes we would like (i.e. works on mobile, unit tests, etc)

**Comment by b-hub** [comment]
Awesome! Thanks for that 👍

**Pull request merged**